LoopKit/InsulinKit/CachedInsulinDeliveryObject+CoreDataClass.swift

```diff
@@ -84,6 +84,35 @@ class CachedInsulinDeliveryObject: NSManagedObject {
             primitiveProgrammedTempBasalRate = NSNumber(value: rate)
         }
     }
+    
+    var modelDuration: Double? {
```

Comment:
In core data, insulin model info is stored as a duration (to add support for Walsh, if desired) and a model type (Walsh, exponential adult, exponential child, or Fiasp)

Reply:
I think we want to be storing the Insulin *type* in HK, and DoseStore. I.e. Fast-Acting, Fiasp, Afrezza, etc. Not specific model of insulin activity. The mapping of insulin type to activity curve would vary based on insulin type and user preferences. I.e. fiasp would map to fiasp curve, and Fast Acting could possibly map to one of the adult or child exponential models, depending on what the user prefers as a Loop setting.